### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by jarede-dev**
fixed grammars and punctuations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,13 +111,13 @@ But we also need to ensure the entire document stays coherent, and I already hav
...
 a-mir-formality is machine-readable and written in PLT redex; MiniRust uses pseudo-code that is not currently machine-readable (but I have ideas :).
 The primary goals of MiniRust are to be precise and human-readable; I would argue that while PLT redex is more precise than the style I use, it does lack in readability when compared with Rust-style pseudo-code.
 I am willing to sacrifice some precision for the significant gain in readability, in particular since I think we can recover this precision with some amount of work.
-And finally, the "operational semantics" layer in a-mir-formality is "not even sketched out yet", so as of now, the projects are actually disjoint.
-If and when a-mir-formality obtains an operational semantics, my hope is that it will be basically the same as MiniRust, just translated into PLT redex.
+And finally, the "operational semantics" layer in a-mir-formality is "not even sketched out yet", so as of now, the projects are actually disjointed.
```
Comment (sanxiyn) [inline README.md:119]: `disjoint` is correct here, see https://www.merriam-webster.com/dictionary/disjoint. It means "having no elements in common".

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Even without deciding what exactly the final memory model will look like, we can
 On the MiniRust language side, the most important concept to understand is that of a *value* and how it relates to *types*.
 Values form a high-level, structural view of data (e.g. mathematical integers); types serve to relate values with their low-level byte-oriented representation.
 Types are essentially just parameters attached to certain operations to define the (de)serialization format.
-Well-formedness of a MiniRust program ensures that expressions and statements satisfy some basic typing discipline, but MiniRust is by design *not* type-safe.
```
Comment (RalfJung) [inline README.md:19]: What's wrong with this? I don't think "well-formedness" needs an article.

File: README.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ All types except for mutable references are `Copy` (let's just imagine we implic
 
 We use `panic!` (and `unwrap` and slice indexing and similar standard Rust operations) to indicate conditions that should always hold; if execution ever panics, that is a bug in the specification.
 
-Our functions are generally pure; they can write to mutable reference but we can consider this to be implemented via explicit state passing.
+Our functions are generally pure; they can write to mutable reference, but we can consider this to be implemented via explicit state passing.
```
Comment (RalfJung) [inline README.md:36]:
```suggestion
Our functions are generally pure; they can write to mutable references, but we can consider this to be implemented via explicit state passing.
```

File: README.md [hunk 4]
```diff
@@ -111,13 +111,13 @@ But we also need to ensure the entire document stays coherent, and I already hav
...
 a-mir-formality is machine-readable and written in PLT redex; MiniRust uses pseudo-code that is not currently machine-readable (but I have ideas :).
 The primary goals of MiniRust are to be precise and human-readable; I would argue that while PLT redex is more precise than the style I use, it does lack in readability when compared with Rust-style pseudo-code.
 I am willing to sacrifice some precision for the significant gain in readability, in particular since I think we can recover this precision with some amount of work.
-And finally, the "operational semantics" layer in a-mir-formality is "not even sketched out yet", so as of now, the projects are actually disjoint.
-If and when a-mir-formality obtains an operational semantics, my hope is that it will be basically the same as MiniRust, just translated into PLT redex.
+And finally, the "operational semantics" layer in a-mir-formality is "not even sketched out yet", so as of now, the projects are actually disjointed.
+If and when a-mir-formality obtains an operational semantics, I hope that it will be basically the same as MiniRust, just translated into PLT redex.
```
Comment (RalfJung) [inline README.md:120]: There's nothing *wrong* with the old sentence here, is there? I think I prefer the cadence of "my hope is" over "I hope" here.

**Comment by RalfJung** [comment]
@jarede-dev thanks for your PR! However, there were some reviews that requested changes and/or clarification. Could you take a look and reply or adjust your PR?

**Comment by RalfJung** [comment]
I pushed some of this in 48b77f3.

**Pull request closed without merging**